src/pudl/ferc_to_sqlite/cli.py

```diff
@@ -171,6 +171,12 @@ def main():  # noqa: C901
         raise_on_error=True,
     )
 
+    # Workaround to reliably getting full stack trace
```

Comment:
Agree that it's janky but I am not too worried about this jank creeping in here - unlikely to spread out throughout the codebase since it's at the user-interaction level, and so nothing depends on this.

Reply:
Sounds good 👍